### ISSUE TITLE
[Frontend] Remove "lvef40" from the Website

### ISSUE THREAD

**Issue by NameAtirut** [biodatlab/multitask-vision-ecg#28]
We have now removed the `lvef40` model from the API. As a result, a random value is returned for this field on the website (as shown in the attached image).

My suspect is that lines 43 and 44 of this file are the cause 
https://github.com/biodatlab/multitask-vision-ecg/blob/dcfde0f7f05e60d8c238f2eccf258f758d1fe82b/app/frontend/libs/mirage.js#L43-L44

I'm not entirely sure if simply removing `lvef40` from the list will solve this or affect anything else.

![image](https://github.com/biodatlab/multitask-vision-ecg/assets/81370547/2319153d-4739-44aa-b212-6c8d9519f945)

**Comment by titipata** [comment]
@bluenex can you take a quick look on this issue?

**Comment by bluenex** [comment]
Sure, will check by today.

**Comment by bluenex** [comment]
@NameAtirut if you mean https://hatai-ai.pages.dev/ecg, that is the test site that doesn't connect to the actual backend yet, so it generates random values regardless of the model.

@titipata I don't recall we deploy this project under your domain name (*.biodatlab.com), have we deployed it somewhere?

**Comment by titipata** [comment]
@bluenex We want to run the demo locally. Not sure where we can adjust it to use the local API. I guess the frontend already renders the key-value that we send it right?

**Comment by bluenex** [comment]
@titipata correct, if the backend is up at `localhost:8000` and there is no `.env` containing `NEXT_PUBLIC_ENABLE_MIRAGE=TRUE` in frontend folder, it should render correctly.
